test/common/router/BUILD

```diff
@@ -15,6 +15,7 @@ envoy_package()
 
 envoy_cc_test(
     name = "config_impl_test",
+    copts = ["-UFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION"],
```

Comment:
How would this end up being defined normally?

Reply:
Hmm, it's probably redundant. If this copt gets overrides by a CLI to define it, then yeah, it's not needed.

Reply:
hmm... is there a way to explicitly undefine it so that even if someone passes in a --copt=-DFUZZING_... to build a fuzz target, that the test would be executed with the macro undefined? researching bazel... but wondering if you know

Reply:
I'm not familiar with a way to do that, unless we wrap the test in a shell scrip that squashes it.

Reply:
Honestly I'm going to make that change in this PR now and test against the OSS-Fuzz build. I feel pretty hesitant removing the fuzzing build macro from our OSS-Fuzz script.